vortex-ffi/cinclude/vortex.h

```diff
@@ -167,6 +173,13 @@ typedef struct vx_file_scan_options {
   int split_by_row_count;
 } vx_file_scan_options;
 
+/**
+ * Options supplied for creating a file.
+ */
+typedef struct vx_file_create_options {
```

Comment:
Similarly, I would just take `const char *` as an argument to the function for now, instead of wrapping in an options struct that you can't really evolve anyway

Reply:
why not, this would give forwards compact no?